src/EngineManager.php

```diff
@@ -107,34 +108,34 @@ protected function defaultAlgoliaHeaders()
     }
 
     /**
-     * Create an MeiliSearch engine instance.
+     * Create an Meilisearch engine instance.
      *
-     * @return \Laravel\Scout\Engines\MeiliSearchEngine
+     * @return \Laravel\Scout\Engines\MeilisearchEngine
      */
     public function createMeilisearchDriver()
     {
-        $this->ensureMeiliSearchClientIsInstalled();
+        $this->ensureMeilisearchClientIsInstalled();
 
-        return new MeiliSearchEngine(
-            $this->container->make(MeiliSearch::class),
+        return new MeilisearchEngine(
+            $this->container->make(MeilisearchClient::class),
             config('scout.soft_delete', false)
         );
     }
 
     /**
-     * Ensure the MeiliSearch client is installed.
+     * Ensure the Meilisearch client is installed.
      *
      * @return void
      *
      * @throws \Exception
      */
-    protected function ensureMeiliSearchClientIsInstalled()
+    protected function ensureMeilisearchClientIsInstalled()
     {
-        if (class_exists(MeiliSearch::class)) {
+        if (version_compare(Meilisearch::VERSION, '0.26.1') >= 0) {
```

Comment:
@driesvints I added this version constraint, so that tests are still possible. As soon as meilisearch and meilisearch-php hit v1. we could set this to `^1.0` only

Reply:
upgraded the version to v1 now so this is rdy for when meilisearch-php v1 is released